ipapython/admintool.py

```diff
@@ -317,7 +317,7 @@ def log_failure(self, error_message, return_value, exception, backtrace):
             # ipa-server-install.
             return
         message = "The %s command failed." % self.command_name
-        if self.log_file_initialized and return_value != 2:
+        if self.log_file_initialized and return_value != SERVER_NOT_CONFIGURED:
```

Comment:
Looks good. Please drop the subsequent comment about magic numbers since the name is being used directly.

Reply:
Done, thanks!